mne/tests/test_coreg.py

```diff
@@ -282,15 +282,17 @@ def test_get_mni_fiducials():
 
 
 @testing.requires_testing_data
-@pytest.mark.parametrize('scale_mode,ref_scale,grow_hair,fiducials', [
-    (None, [1., 1., 1.], 0., None),
-    (None, [1., 1., 1.], 0., 'estimated'),
-    (None, [1., 1., 1.], 2., 'auto'),
-    ('uniform', [1., 1., 1.], 0., None),
-    ('3-axis', [1., 1., 1.], 0., 'auto'),
-    ('uniform', [0.8, 0.8, 0.8], 0., 'auto'),
-    ('3-axis', [0.8, 1.2, 1.2], 0., 'auto')])
-def test_coregistration(scale_mode, ref_scale, grow_hair, fiducials):
+@pytest.mark.parametrize(
+    'scale_mode,ref_scale,grow_hair,fiducials,fid_match', [
+        (None, [1., 1., 1.], 0., None, 'nearest'),
+        (None, [1., 1., 1.], 0., 'estimated', 'nearest'),
+        (None, [1., 1., 1.], 2., 'auto', 'nearest'),
+        ('uniform', [1., 1., 1.], 0., None, 'nearest'),
+        ('3-axis', [1., 1., 1.], 0., 'auto', 'nearest'),
+        ('uniform', [0.8, 0.8, 0.8], 0., 'auto', 'nearest'),
+        ('3-axis', [0.8, 1.2, 1.2], 0., 'auto', 'nearest')])
```

Comment:
These are all `'nearest'`, one needs to be `'matched'`, right?

Reply:
🤦🏻